va-gov/layouts/home.html

```diff
@@ -223,4 +223,19 @@ <h4 class="homepage-news-story-title"><a href=https://www.blogs.va.gov/VAntage/4
 
 </main>
 
+<script type="text/javascript">
+  var hubLinksList = document.getElementsByClassName('hub-links-list');
+
+  for (var i=0; i < hubLinksList.length; i++) {
+    hubLinksList[i].addEventListener('click', function(e) {
+      var linkData = e.target.dataset;
+
+      window.dataLayer.push({
+        event: 'nav-zone-one',
+        'nav-path': linkData.navPath,
+      });
+    });
+  }
+</script>
+
```

Comment:
🔥 🔥 🔥 

Can we use `recordEvent` here instead of `window.dataLayer.push`? It's just an alias function.

This conflicts with the work @jbalboni is doing with the homepage, so you may need to look into the changes over at https://github.com/department-of-veterans-affairs/vets-website/pull/8848.

Reply:
Just saw the conflict. Yes I can use the recordEvent. Thanks